package.json

```diff
@@ -4,7 +4,9 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint '**/*js'",
+    "lint:fix": "prettier-eslint --write '**/*.{js,jsx}' "
```

Comment:
Lol did you just copy my repo and you didn't even star 😢 . Please remove the .jsx this is backend

Reply:
Lolz, so sorry about the star.  I left the .jsx because we will be using react for the front end but I will remove it and probably add it back when we start the front end. is that cool by you?

Reply:
front-end will be on a different repo

Reply:
Ok. I have  Implemented feedback